keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -56,8 +56,7 @@ private fun ClusterSpec.resolveLaunchConfiguration(region: SubnetAwareRegionSpec
       ?: defaults.launchConfiguration?.iamRole
       ?: LaunchConfiguration.defaultIamRoleFor(moniker.app)),
     keyPair = checkNotNull(overrides[region.name]?.launchConfiguration?.keyPair
-      ?: defaults.launchConfiguration?.keyPair
-      ?: LaunchConfiguration.defaultKeyPairFor(locations.account, region.name)),
+      ?: defaults.launchConfiguration?.keyPair),
```

Comment:
This does not assume the hard-coded default key pair format now, and instead lets `LaunchConfigurationResolver` figure that out based on information reported by clouddriver.